src/main/scala/firrtl/stage/transforms/WrappedTransform.scala

```diff
@@ -0,0 +1,33 @@
+// See LICENSE for license details.
+
+package firrtl.stage.transforms
+
+import firrtl.Transform
+
+/** A [[firrtl.Transform]] that "wraps" a second [[firrtl.Transform Transform]] to do some work before and after the
+  * second [[firrtl.Transform Transform]].
+  *
+  * This is intended to synergize with the [[firrtl.options.DependencyManager.wrappers]] method.
+  * @see [[firrtl.stage.transforms.CatchCustomTransformException]]
```

Comment:
IntelliJ says the CatchCustomTransformException link here is broken.
I think it should be plural CatchCustomTransformExceptions

Reply:
Good catch. Fixed in the rebase.